lab07/tests/q1.py

```diff
@@ -32,7 +32,7 @@
                                        'locked': False},
                                    {   'code': ">>> trump['time'].dtype == "
                                                "np.dtype('<M8[ns]')\n"
-                                               'False',
```

Comment:
Sorry, I did not intend this commit to be included in the pull request... not exactly how that happened because the pull request was created before this commit!